src/CMakeLists.txt

```diff
@@ -204,34 +202,54 @@ ELSE()
 ENDIF()
 
 # Specify sha1 implementation
-IF (USE_SHA1DC)
+IF ( SHA1_BACKEND STREQUAL "AUTO" )
+	IF ( OPENSSL_FOUND )
+		SET(SHA1_BACKEND "OpenSSL")
+	ELSEIF (WIN32 AND NOT MINGW)
+		SET(SHA1_BACKEND "SHA1_WIN32")
+	ELSEIF (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
Nitpick: ISTR the `${}` syntax is a leftover of an old era where CMake didn't automatically evaluate expressions in `IF` statements. Personally, I find it inconsistent to only use that syntax for `CMAKE_SYSTEM_NAME`, while all other expressions don't.

Reply:
Noted, will fix tomorrow